packages/rspack/etc/api.md

```diff
@@ -9822,10 +9822,12 @@ type ResolveRequest = BaseResolveRequest & Partial<ParsedIdentifier>;
 
 // @public (undocumented)
 class ResolverFactory {
```

Comment:
@h-a-n-a is this real public api? how users can access this api?

Reply:
This is not according to the generated `api.md`